tools/cmake/cores/Cortex-M33FE-NS.cmake

```diff
@@ -7,11 +7,11 @@ if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
         "-mthumb"
         "-mfpu=fpv5-sp-d16"
         "-mfloat-abi=softfp"
-        "-march=armv8-m.main+dsp"
+        "-mcpu=cortex-m33+dsp"
     )
 elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
     list(APPEND common_options
-        "-mcpu=cortex-m33"
+        "-mcpu=cortex-m33+dsp"
```

Comment:
Unlike for GCC, this is perfectly fine according to the docs, as `+X` and `+noX` are always available. But if you do want to say `+dsp` to be explicit, why not `+fp` too?

I would imagine ARMC6's defaults are the same as GCC's so might be best to keep it the same as GCC.

Reply:
fixed, I only followed for now what we had (separate fpu settings for both toolchains). To keep both toolchains the same.

Reply:
Yeah, it's not clear from docs how `+fp`/`+nofp` (or lack of and hence default) and `-mfpu` interact. Maybe don't mess with whatever we know/think works from original python stuff.